src/ugeneui/src/project_support/ProjectImpl.cpp

```diff
@@ -71,6 +71,23 @@ void ProjectImpl::makeClean() {
     assert(!isTreeItemModified());
 }
 
+void ProjectImpl::sl_removeAllrelationsWithObject(GObject* obj) {
+    QList<GObject*> allObjs;
+    for (Document *d : qAsConst(getDocuments())) {
+        allObjs << d->getObjects();
+    }
+    for (GObject* object : qAsConst(allObjs)) {
+        if (object != obj) {
+            const QList<GObjectRelation> relationList = object->getObjectRelations();
+            for (const GObjectRelation& rel : relationList) {
```

Comment:
Approved with no qAsConst this time (will fix it by myself). Will be a blocker next time.